main/FAB/index.tsx

```diff
@@ -46,10 +53,12 @@ function FloatingActionButtons<Item extends FABItem = FABItem>({
 }: FABProps<Item> & {
   theme: DoobooTheme;
 }): ReactElement {
+  const GAP = 15;
   const {FAB, FABItem, buttonSize = 'large', iconSize = 24} = styles ?? {};
 
-  const spinValue = useRef(new Animated.Value(0));
-  const positionValue = useRef(new Animated.Value(0));
+  const spinValue = useRef<Animated.Value>(new Animated.Value(0));
+  const positionValue = useRef<Animated.Value>(new Animated.Value(0));
+  const FABHeight = useRef<number>(0);
```

Comment:
I suggest you to write like this.

```ts
const spinValue = useRef(new Animated.Value(0));
const positionValue = useRef(new Animated.Value(0));
const FABHeight = useRef(0);
```

See [this](https://github.com/dooboolab/dooboo-ui/pull/110#discussion_r706622897) for details.

Reply:
oh, I see. 
thank you for pointing it out!

Reply:
@yujong-lee 
I reflected this review thanks.